interpolation/interpolation.go

```diff
@@ -0,0 +1,116 @@
+// Package interpolation provides methods to interpolate user defined variables and robo tasks.
+package interpolation
```

Comment:
How do you feel about changing the name to `interpolate` and then the functions to `Vars()`, `Tasks()` etc.. 
Not a huge deal btw, just follows Go's naming conventions :)

Reply:
You're right. I had it in the config package first and moved it afterwards. Will rename that one quickly 👍